nautobot/extras/tests/test_views.py

```diff
@@ -4069,17 +4072,7 @@ def test_list_objects_with_constrained_permission(self):
         self.assertNotIn(instance2.assigned_object.get_absolute_url(), content, msg=content)
 
 
-class RelationshipTestCase(
-    ViewTestCases.CreateObjectViewTestCase,
-    ViewTestCases.DeleteObjectViewTestCase,
-    ViewTestCases.EditObjectViewTestCase,
-    ViewTestCases.BulkDeleteObjectsViewTestCase,
-    ViewTestCases.GetObjectViewTestCase,
-    ViewTestCases.GetObjectChangelogViewTestCase,
-    ViewTestCases.ListObjectsViewTestCase,
-    RequiredRelationshipTestMixin,
-    ViewTestCases.BulkEditObjectsViewTestCase,
-):
+class RelationshipTestCase(ViewTestCases.PrimaryObjectViewTestCase):
```

Comment:
as far as I can tell, `RequiredRelationshipTestMixin` doesn't provide anything this test actually needed, and we were missing `GetObjectNotesViewTestCase` mixin, so changing to `PrimaryObjectViewTestCase` gives us net +1 test here.